### PR TITLE
Add example of no. of entities for scorecards

### PR DESCRIPTION
**Pull request by Matanga1-2**
# Description

Added example scorecard that count number of entities related.

## Updated docs pages
- scorecards (

### CODE REVIEW

File: docs/promote-scorecards/examples.md [hunk 1]
```diff
@@ -97,4 +97,50 @@ We can define a scorecard that checks that all of our services have a related do
     }
   ]
 }
+```
+
+### Validate number of related entities
+
+To effectively track the `Team` deployment frequency and set rules based on it, simply checking the `Deployment` relation to the Team is not enough—we need to know the exact number of deployments. To achieve this, we will:
+
+- Create an [aggregation property](/build-your-software-catalog/customize-integrations/configure-data-model/setup-blueprint/properties/aggregation-property.md) to count the number of related entities (e.g., the number of deployments in the past week).
```
Comment (hadar-co) [inline docs/promote-scorecards/examples.md:106]: Create an aggregation prop on which blueprint? This example needs a bit of an intro of the scenario

File: docs/promote-scorecards/examples.md [hunk 2]
```diff
@@ -97,4 +97,50 @@ We can define a scorecard that checks that all of our services have a related do
     }
   ]
 }
+```
+
+### Validate number of related entities
```
[the remark this answers is not in the document]
Comment (Matanga1-2) [reply docs/promote-scorecards/examples.md:102]: Agree, fixing

File: docs/promote-scorecards/examples.md [hunk 3]
```diff
@@ -97,4 +97,50 @@ We can define a scorecard that checks that all of our services have a related do
...
+
+To effectively track the `Team` deployment frequency and set rules based on it, simply checking the `Deployment` relation to the Team is not enough—we need to know the exact number of deployments. To achieve this, we will:
```
[the remark this answers is not in the document]
Comment (Matanga1-2) [reply docs/promote-scorecards/examples.md:104]: Agree, fixing and adjusting to "service"

File: docs/promote-scorecards/examples.md (hunk 1 above)
Comment (Matanga1-2) [reply docs/promote-scorecards/examples.md]: Agree, added

File: docs/promote-scorecards/examples.md [hunk 4]
```diff
@@ -97,4 +97,50 @@ We can define a scorecard that checks that all of our services have a related do
     }
   ]
 }
+```
+
+### DORA metrics based on number of deployments
+
+To assess the deployment frequency of a `Service`, simply checking the `Deployment` relation is not enough—we need to know the exact number of deployments. To achieve this, we will:
```
Comment (hilakashai) [inline docs/promote-scorecards/examples.md:104]:
```suggestion
To assess the deployment frequency of a `Service`, simply checking the `Deployment` relation is not enough — we need to know the exact number of deployments. To achieve this, we will:
```

File: docs/promote-scorecards/examples.md [hunk 5]
```diff
@@ -97,4 +97,50 @@ We can define a scorecard that checks that all of our services have a related do
     }
   ]
 }
+```
+
+### DORA metrics based on number of deployments
+
+To assess the deployment frequency of a `Service`, simply checking the `Deployment` relation is not enough — we need to know the exact number of deployments. To achieve this, we will:
```
Comment (hadar-co) [inline docs/promote-scorecards/examples.md:104]:
```suggestion
To assess the deployment frequency of a `service`, simply checking the `deployment` relation is not enough — we need to know the exact number of deployments. To achieve this, we can:
```

File: docs/promote-scorecards/examples.md [hunk 6]
```diff
@@ -97,4 +97,50 @@ We can define a scorecard that checks that all of our services have a related do
...
+
+- Add an [aggregation property](/build-your-software-catalog/customize-integrations/configure-data-model/setup-blueprint/properties/aggregation-property.md) to the `Service` blueprtint that counts the number of related `Deployment` entities.
```
Comment (hadar-co) [inline docs/promote-scorecards/examples.md:106]:
```suggestion
- Add an [aggregation property](/build-your-software-catalog/customize-integrations/configure-data-model/setup-blueprint/properties/aggregation-property.md) to the `service` blueprint that counts the number of related `deployment` entities.
```

File: docs/promote-scorecards/examples.md [hunk 7]
```diff
@@ -97,4 +97,50 @@ We can define a scorecard that checks that all of our services have a related do
...
+- Add a scorecard with a rule based on that property.
```
Comment (hadar-co) [inline docs/promote-scorecards/examples.md:107]:
```suggestion
- Add a scorecard with a rule based on the new aggregation property:
```

**Review by hadar-co — APPROVED** [review]

**Pull request merged**